src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -176,7 +182,9 @@ public String signature() throws IOException {
                 )
             ).value()
         ).asString();
-        if (!RtTransaction.SIGN.matcher(sign).matches()) {
+        // @checkstyle MagicNumber (1 line)
+        if (sign.length() % 4 != 0
```

Comment:
@carlosmiranda shouldn't we check the specific number specified in the whitepaper? Not sure about it though, I also think it should be a multiple of 4 since thta's what should be a base64 string, but I would be interested in your rational before accepting this.

Reply:
@victornoel I already explained it in the description: https://github.com/zold-io/java-api/pull/57#issue-205029360

Reply:
@carlosmiranda when in my review I proposed to check the length here, I was expecting you to make the explanation in the description obsolete.
I don't understand the rationale behind checking it is a multiple of 4 instead of checking that the length is the correct one (384 characters in the paper). Note that I am not saying it MUST be exactly 384 characters, but I want to ensure we agree on what is happening here :)

Reply:
@victornoel I'm checking that it's a valid base64 string. That means the given regex + multiple of 4 length.

>I was expecting you to make the explanation in the description obsolete.

Well don't be passive aggressive about it. ;)

Should I change it to comply with the whitepaper spec? Right now we are already stricter than the Ruby implementation which I am basing this on.

Reply:
@victornoel no worries, I was just confused as to whether you were asking a change or a clarification.

I'll go ahead with enforcing the 684 characters, it's fairly trivial to do.